index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>MICROVERSE ASSIGNMENT</title>
+    <link rel="stylesheet" href="/styles/style.css" />
+    <link rel="stylesheet" href="/styles/nav.css" />
+    <link rel="stylesheet" href="/styles/content.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.0/css/all.min.css" integrity="sha512-BnbUDfEUfV0Slx6TunuB042k9tuKe3xrD6q4mg5Ed72LTgzDIcLPxg6yI2gcMFRyomt+yJJxE+zJwNmxki6/RA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  </head>
+  <body>
+    <div class="nav">
+      <!-- Left nav -->
+      <!-- <div class="d-inline-flex p-2"></div> -->
+      <div class="nav-left">
+        <span class="nav-image-container">
+          <img src="images/menu.png" alt="img" />
+        </span>
+        <span class="nav-image-container">
+          <img src="images/sec logo.jpg" alt="img" />
+        </span>
+      </div>
```

Comment:
good job so far but please kindly used a nav tags for the nav bar so that it should be semantically proper